backends/mongo.go

```diff
@@ -29,6 +30,7 @@ type Mongo struct {
 	Conn             *mongo.Client
 	disableSuperuser bool
 	hasher           hashing.HashComparer
+	Tls              bool
```

Comment:
```suggestion
	withTLS              bool
```
That would be consistent with how we call the option in `http` and `jwt` backends.